webapp/src/linterChecks/Signe.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Looks like this file is from your last PR. I think you branched off your own instead of off of the `main` branch.

Because I did a squash & merge(I believe) git now thinks this is new again

Blue is main, purple is REC-16-hasura-metadta, pink (magenta?) is REC-32_basic-nav an orange is REC-22_machine-setup in this screenshot.

What we want is purple, pink, and orange to all branch off of blue now. You can do that by either rebasing so the feature branch is on top of main OR you can merge main into yours. HOWEVER, if you do a merge we MUST squash & merge because of how I setup github. I'd recommend the rebase if you're comfortable with that workflow (or want to learn it). For my projects the merge works but is often unideal (changes project to project though!)

If you end up needing a walkthrough of it we should have a call with Arsh (and optionally Akshay).
![image](https://user-images.githubusercontent.com/485809/149818129-46949835-e5ea-46e2-8bf3-56026abc7955.png)